### ISSUE TITLE
Support time series anomaly algorithms

### ISSUE THREAD

**Issue by vejadu** [dotnet/machinelearning#163]
### Issue

Would love to have/add support for time series forecasting/anomaly detection. There's some pretty common scenarios such as

- different types of forecasting such as HW (TES and DES)
- online aspect of these algorithms (performant; so no retraining/evaluation is necessary to be integrated in a streaming fashion)
- PScore/ZScore/threadhold evaluation for anomalies

**Comment by GalOshri** [comment]
Absolutely! This is on the [roadmap](https://github.com/dotnet/machinelearning/blob/436700aadf615e7f05a22925476cc441c63a919d/ROADMAP.md). Are there any specific scenarios you are trying to enable first?

**Comment by vejadu** [comment]
Effectively HW TES w/ PScore thresholding w/ auto tuning (alpha/beta/gamma) and retraining every N intervals w/ the ability to serialize/deserialize the model to pick up where we left off in the forecasting. (it's what I currently have working in my current usage scenario.)

**Comment by veikkoeeva** [comment]
Adding here for the benfit of others: https://github.com/Spreads/Spreads.

**Comment by abgoswam** [comment]
DRI RESPONSE : Adding support for time series is being tracked using issue #978. Planning to close this issue (duplicate of #978)
